app-backup/buttersink/buttersink-0.6.8.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="rsync-like utility for btrfs snapshots"
+HOMEPAGE="https://github.com/AmesCornish/buttersink"
+
+LICENSE="GPL-3"
+SLOT=0
+
+# tests require network access
+RESTRICT="test"
```

Comment:
The problems with the tests go beyond the network access. Just doing the local tests would require root access, and would also cause sandbox issues (even creating a test btrfs filesystem in a loop device, I think you'd still need addread/addwrite on /dev/loop*). To test the network functionality you'd also need an SSH server you could log into as root for the SSH backend, or likewise a remote S3 for that backend. The makefile would also need some patching.